lib/log/level.js

```diff
@@ -12,6 +22,10 @@ const LEVELS_VALUE = {
 	'SILLY':   5
 };
 
+/**
+ * Return the level based on the argument ( also transform it to upperCase)
+ * @param {*} level
```

Comment:
the @param comment for this function should specify the data type for the argument